hack/determine_install_upgrade_version_test.py

```diff
@@ -0,0 +1,95 @@
+import unittest
+import rpmUtils.miscutils as rpmutils
+from determine_install_upgrade_version import *
+
+class TestPackage(object):
+	def __init__(self, version, release, vra):
+		self.version = version
+		self.release = release
+		self.vra = vra
+
+class GetMatchingVersionTestCase(unittest.TestCase):
+	"Test for `determine_install_upgrade_version.py`"
+
+	def setUp(self):
+		test_pkgs = ["origin-1.2.0-0.el7.x86_64",
```

Comment:
Prefer not to use global state, list the specific things you need in each test as necessary